package.json

```diff
@@ -96,7 +96,7 @@
     "pouchdb-adapter-http": "^7.2.1",
     "pouchdb-core": "^7.2.1",
     "pouchdb-mapreduce": "^7.2.1",
-    "protractor": "^5.4.4",
+    "protractor": "^7.0.0",
```

Comment:
The protractor README says that protractor 7 isn't compatible with Node 8, just 10, 12 and newer. 
https://github.com/angular/protractor#compatibility

I wonder if we'll start having issues once we enable more tests (we have a test suite running Node 8).

Reply:
The same is true for Angular in general. They only support "Angular requires a current, active LTS, or maintenance LTS version of Node.js." Which Node 10 is the most recent maintenance version.